behave/features/steps/specialist_provider.py

```diff
@@ -282,15 +291,67 @@ def step_impl(context):
                                                     f"{CLA_SPECIALIST_REJECTION_OUTCOME_CODES[reject_reason]}']")
     assert outcome_code is not None
     assert outcome_code.get_attribute("title") == CLA_SPECIALIST_REJECTION_OUTCOME_CODES[reject_reason]
+
+
+@step(u'the \'New case\' drop down values are')
+def step_impl(context):
+    # Find the modal container
+    context.modal = context.helperfunc.find_by_css_selector('.modal-dialog')
+    # Inside modal find the form we want to focus on.
+    context.form = context.modal.find_element_by_xpath("//form[@name='split_case_frm']")
+    for row in context.table:
+        label = row['label']
+        value = row['value']
+
+        label_link = context.form.find_element_by_xpath(f"//span[text()='{label}']/../../span/span/div/a")
+        # Clicking this link will automatically focus the input for us to type into
+        label_link.click()
+
+        def wait_for_list_of_values(*args):
+            try:
+                # Try and see when this element is visible in the modal
+                context.form.find_element_by_xpath(f"//li/div[text()='{value}']")
+                return True
+            except NoSuchElementException:
+                return False
+
+        wait = WebDriverWait(context.helperfunc.driver(), 10)
+        wait.until(wait_for_list_of_values, message=f"Could not find any matches for {value} in {label} list")
+        list_item = context.form.find_element_by_xpath(f"//li/div[text()='{value}']")
+        list_item.click()
+
+        # Once list item has been selected, check the anchor contains the correct value
+        label_value = context.form.find_element_by_xpath(f"//span[text()='{label}']"
+                                                         f"/../../span/span/div/a/span[text()='{value}']")
+        assert label_value.text == value
+
+
+@step(u'I enter a comment into the \'New case\' notes textarea')
+def step_impl(context):
+    context.modal = context.helperfunc.find_by_css_selector('.modal-dialog')
+    comment = LOREM_IPSUM_STRING
+    text_area = context.modal.find_element_by_xpath('//textarea[@name="notes"][@placeholder="Enter comments"]')
+    text_area.send_keys(comment)
+    assert text_area.get_attribute("value") == comment
+
+
+@step(u'I select \'{value}\' for the \'Assign\' radio options')
+def step_impl(context, value):
+    context.modal = context.helperfunc.find_by_css_selector('.modal-dialog')
+    modal_input = context.modal.find_element_by_xpath(f"//input[@value="
+                                                      f"'{CLA_SPECIALIST_SPLIT_CASE_RADIO_OPTIONS[value]}']")
+    assert modal_input is not None
+    modal_input.click()
+
+
+@step(u'the new case is visible in the operators dashboard')
+def step_impl(context):
+    table = context.helperfunc.driver().find_element_by_css_selector(".ListTable")
+    table.find_element_by_xpath(f"//a[text()='{CLA_SPECIALIST_CASE_TO_SPLIT}']")
```

Comment:
The step is called `the new case is visible in the operators dashboard` but his this step is checking the original case reference